fish-rust/src/builtins/shared.rs

```diff
@@ -45,7 +45,9 @@ pub const STATUS_CMD_OK: Option<c_int> = Some(0);
 /// The status code used for failure exit in a command (but not if the args were invalid).
 pub const STATUS_CMD_ERROR: Option<c_int> = Some(1);
 
-/// A handy return value for invalid args.
+/// The status code used for invalid arguments given to a command. This is distinct from valid
+/// arguments that might result in a command failure. An invalid args condition is something
+/// like an unrecognized flag, missing or too many arguments, an invalid integer, etc. But
```

Comment:
Is there something missing from the sentence here?

Reply:
fixed